core/src/main/scala/tofu/lift/Unlift.scala

```diff
@@ -34,6 +34,11 @@ object Lift {
     new Lift[F, G] {
       def lift[A](fa: F[A]): G[A] = iso.to(fa)
     }
+
+  def byFunK[F[_], G[_]](fk: F ~> G): Lift[F, G] =
+    new Lift[F, G] {
+      def lift[A](fa: F[A]): G[A] = fk(fa)
+    }
 }
```

Comment:
Let's apply consistent formatting to similar code pieces. I mean use of `override` and blank lines.